infra/modules/ai/cognitiveservices.bicep

```
@@ -82,7 +82,10 @@ param raiPolicies array = [
 
 
 resource deploymentScript 'Microsoft.Resources/deploymentScripts@2023-08-01' = [for raiPolicy in raiPolicies: {
-  name: 'CreateRAIPolicy_${raiPolicy.name}'
+  dependsOn: [
+    roleAssignmentDeploymentScript
+  ]
+  name: 'create_rai_policy__${raiPolicy.name}_${location}'
```

```suggestion
  name: 'create_rai_policy__${raiPolicy.name}_${account.name}'
```

Perhaps we should rename this to account.name? It would be possible to have multiple resources in the same region, while a deployment script is region specific.

---

yep, will add a commit